### PR TITLE
modify ckj weight

### PR DESCRIPTION
**Pull request by ken2403**

### CODE REVIEW

File: lcaonet/model/lcaonet.py [hunk 1]
```diff
@@ -306,7 +306,7 @@ def __init__(
         self.basis_weight = Dense(conv_dim, conv_dim, False, weight_init)
 
         self.f_node = nn.Sequential(
-            Dense(2 * hidden_dim, conv_dim, True, weight_init),
+            Dense(2 * conv_dim, conv_dim, True, weight_init),
```
Comment (ken2403) [inline lcaonet/model/lcaonet.py:309]: bag fix:
fix shape of node embedding

File: lcaonet/model/lcaonet.py [hunk 2]
```diff
@@ -348,37 +348,40 @@ def forward(
         if self.add_valence and valence_mask is None:
             raise ValueError("valence_mask must be provided when add_valence=True")
 
+        x_before = x
+
+        # Transformation of the node vectors
+        x, xk = torch.chunk(self.node_weight(x), 2, dim=-1)
```
Comment (ken2403) [inline lcaonet/model/lcaonet.py:354]: add x_i, and x_j weight.

File: lcaonet/model/lcaonet.py [hunk 3]
```diff
@@ -348,37 +348,40 @@ def forward(
...
+
         # Transformation of the coefficient vectors
         cji = self.f_coeffs(cji)
-        cji, ckj = torch.chunk(cji, 2, dim=-1)
```
Comment (ken2403) [inline lcaonet/model/lcaonet.py:353]: rm ckj weight

File: lcaonet/model/lcaonet.py [hunk 4]
```diff
@@ -348,37 +348,40 @@ def forward(
...
 
         # cutoff
         if cutoff_w is not None:
             rb = rb * cutoff_w.unsqueeze(-1)
 
         # --- Threebody Message-passing ---
-        ckj = ckj[edge_idx_kj]
-        ckj = F.normalize(ckj, dim=-1)
```
Comment (ken2403) [inline lcaonet/model/lcaonet.py:361]: rm normalize of coefficient

File: lcaonet/model/lcaonet.py [hunk 5]
```diff
@@ -348,37 +348,40 @@ def forward(
...
-        three_body_w = three_body_orbs * xk
+        xk = torch.sigmoid(xk[tri_idx_k])
+        three_body_w = three_body_w * xk
         three_body_w = self.f_three(scatter(three_body_w, edge_idx_ji, dim=0, dim_size=rb.size(0)))
 
         # threebody orbital information is injected to the coefficient vectors
         cji = cji + cji * three_body_w.unsqueeze(1)
 
-        # --- Twobody Message-passing ---
-        cji = F.normalize(cji, dim=-1)
```
Comment (ken2403) [inline lcaonet/model/lcaonet.py:381]: rm normalize of coefficeient

File: test/model/test_lcaonet.py [hunk 6]
```diff
@@ -224,6 +224,20 @@ def test_LCAOOut(
 
     assert out.size() == (n_batch, out_dim)
 
+    if batch:
```
Comment (ken2403) [inline test/model/test_lcaonet.py:227]: add output value test.
scatter and sum, mean test
